test/e2e/e2e_test.go

```diff
@@ -60,6 +68,348 @@ import (
 	"sigs.k8s.io/descheduler/test"
 )
 
+func isClientRateLimiterError(err error) bool {
+	return strings.Contains(err.Error(), "client rate limiter")
+}
+
+func deschedulerSA() *v1.ServiceAccount {
+	return &v1.ServiceAccount{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "descheduler-sa",
+			Namespace: "kube-system",
+		},
+	}
+}
+
+func deschedulerClusterRole() *rbacv1.ClusterRole {
```

Comment:
is there a reason that we're duplicating these, as opposed to using [this](https://github.com/kubernetes-sigs/descheduler/blob/master/kubernetes/base/rbac.yaml)?

Reply:
Good point. We need to reconcile. I prefer the Go version of all the manifests so we can skip the unmarshaling part. I wonder if it would make sense to create e.g. `hack/manifests2go.sh` that will do the trick and gets run right before e2e.

Reply:
@a7i SA, CR and CRB are now applied from `kubernetes/base/rbac.yaml`